stacks/java-maven/devfile.yaml

```diff
@@ -16,7 +16,8 @@ starterProjects:
 components:
   - name: tools
     container:
-      image: quay.io/eclipse/che-java11-maven:next
+      image: registry.access.redhat.com/ubi8/openjdk-11:latest
+      command: ["tail", "-f", "/dev/null"]
```

Comment:
Can we use UDI instead for the sake of unification https://github.com/devfile/developer-images ?

Reply:
we use this image pretty much for all the devfile v2 getting started samples in Eclipse Che and the workspace startup is pretty fast. Also the udi image is pat of the devfile org, so IMO for the sake of unification we should use it in the devfile samples. @l0rd @slemeur wdyt?

Reply:
I would prefer to keep the use of UDI images as a different discussion.

This PR is meant to address the issue of the use of outdated images.

Swithing to UDI is completely different issue.

Reply:
well, if the image is updated anyways why not do it in one go if we can agree on having the UDI images used across the samples? We are having multiple issues (this PR is a good example) when the images in devfiles are terminated which resulted in broken samples on OpenShift for Eclipse Che / CRW. if we can have a unified approach implemented with UDI it would improve the maintainability dramatically

Reply:
> if we can agree on having the UDI images used across the samples? 

https://github.com/devfile/api/issues/616#issuecomment-1218150338